corehq/apps/data_interfaces/views.py

```diff
@@ -498,7 +500,8 @@ def post(self, request, *args, **kwargs):
             mode,
             self.domain,
             self.request.couch_user,
-            form_ids_or_query_string
+            form_ids_or_query_string,
+            self.request.can_access_all_locations
         )
```

Comment:
This should probably check permissions on the forms being updated before calling the task.  How does that happen currently?